sprucelabsai/sprucebot-skills-kit
# SB-889 - sequelize settings and model/migrations examples

[SB-889](https://jira.sprucelabs.ai/jira/browse/SB-889)

@sprucelabsai/engineers

## Description 
Adds a sequelize adapter and automatic model ingestion and migrations

## Type

## Steps to Test or Reproduce
Follow the Readme https://github.com/sprucelabsai/sprucebot-skills-kit/pull/61/files#diff-04c6e90faac2675aa89e2176d2eec7d8R15

This should be tested in tandem with https://github.com/sprucelabsai/sprucebot-skills-kit-server/pull/5 and https://github.com/sprucelabsai/com-sprucebot-api/pull/152 and https://github.com/sprucelabsai/sprucebot-node/pull/11
